app/javascript/components/search/controls/download/DownloadSelectionModal.js

```diff
@@ -41,8 +41,12 @@ export default function DownloadSelectionModal({ studyAccessions, azulFileInfo,
     render bulk download table for SCP & HCA studies
    */
   function renderFileTables(result=[]) {
-    setSelectedBoxes(newSelectedBoxesState(result, SCP_COLUMNS))
-    setDownloadInfo(result)
+    const partitionedResults = partition(result, ['studySource', 'SCP'])
+    const scpFileInfo = partitionedResults[0]
+    const azulFileInfo = partitionedResults[1]
```

Comment:
```suggestion
    const [scpFileInfo, azulFileInfo]  = _partition(result, ['studySource', 'SCP'])
```
using a destructured assignment for clarity/conciseness

Reply:
aah I wasn't sure if that worked in JS - nice.

Reply:
39fe8e0